apps/experiments/views/experiment.py

```diff
@@ -514,29 +511,38 @@ def single_experiment_home(request, team_slug: str, experiment_id: int):
         deployed_version = experiment.default_version.version_number
 
     bot_type_chip = None
-    if pipeline := experiment.pipeline:
-        bot_type_chip = Chip(label=f"Pipeline: {pipeline.name}", url=pipeline.get_absolute_url())
-    elif assistant := experiment.assistant:
-        bot_type_chip = Chip(label=f"Assistant: {assistant.name}", url=assistant.get_absolute_url())
+    if active_tab == "experiments":
+        if pipeline := experiment.pipeline:
+            bot_type_chip = Chip(label=f"Pipeline: {pipeline.name}", url=pipeline.get_absolute_url())
+        elif assistant := experiment.assistant:
+            bot_type_chip = Chip(label=f"Assistant: {assistant.name}", url=assistant.get_absolute_url())
+    elif active_tab == "chatbots" and experiment.pipeline:
+        bot_type_chip = Chip(label=f"Pipeline: {experiment.pipeline.name}", url=experiment.pipeline.get_absolute_url())
+
+    context = {
+        "active_tab": active_tab,
+        "bot_type_chip": bot_type_chip,
+        "experiment": experiment,
+        "user_sessions": user_sessions,
+        "platforms": available_platforms,
+        "platform_forms": platform_forms,
+        "channels": channels,
+        "available_tags": [tag.name for tag in experiment.team.tag_set.filter(is_system_tag=False)],
+        "experiment_versions": experiment.get_version_name_list(),
+        "deployed_version": deployed_version,
+        **_get_events_context(experiment, team_slug),
+        **_get_routes_context(experiment, team_slug),
```

Comment:
`_get_events_context` and `_get_routes_context` can also be removed when it is the chatbot tab

Reply:
```suggestion
```